continuous_integration/test_script.sh

```diff
@@ -25,4 +30,9 @@ else
     nosetests -s sklearn
 fi
 
+# Is directory still empty ?
```

Comment:
the lines above about "make test" and "make test-coverage" are not really applicable any more, right?